transmit-lib/src/cmr/transmit/generic_documents.clj

```diff
@@ -22,13 +22,14 @@
   ([conn provider-id]
    {:pre [conn provider-id]}
    (println "in concept ingest url")
-   (format "%s/generics/%s" (conn/root-url conn) (codec/url-encode provider-id)))
+   (format "%s/generics/%s/%s" (conn/root-url conn) (codec/url-encode provider-id) "nativeId"))
```

Comment:
I don't understand why we are passing in a hardcoded `nativeId` in the routes here and below.

Reply:
Thats because we are in the process of putting native id back in.  currently in the feature/generic-document-pipeline native id is in metadata_db but not in ingest or the indexer app.  So I put it in there so that I could ingest 1 document so that I can continue on with my work.  I can remove it.  Thomas is working on putting in the native id back in the rest of the code, so by the end of this sprint this code will be removed.

Reply:
ok. It would be nice to have a comment with TODO.